superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -159,9 +159,23 @@ export default class ResultSet extends React.PureComponent<
     this.handleExploreBtnClick = this.handleExploreBtnClick.bind(this);
   }
 
-  componentDidMount() {
+  async componentDidMount() {
     // only do this the first time the component is rendered/mounted
     this.reRunQueryIfSessionTimeoutErrorOnMount();
+
+    const response = await makeApi({
+      method: 'GET',
+      endpoint: '/api/v1/dataset',
```

Comment:
do we need to pass in the user id like below on line 342? What's the difference in the api response if you don't?